### DISCUSSION TITLE
Postcss mantine questions

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#4858] [Q&A]
1) colorSchemeMixin
now it generates `[data-mantine-color-scheme='dark'] xxx`
Is there a way to change the value of data attribute to another selector or at least to another data attribute.
It would be very helpful for those people like me who uses approved tailwind config and don't have easy way to change 
`darkMode` prop
For example I have 
```ts
  darkMode: ['class', '[data-color-scheme="dark"]'],
```
and for example a codebase with different relations on this data-attribute not only in styles.
FR: add an optional plugin option in [postcss-preset-mantine](https://github.com/mantinedev//blob/master/src/preset.ts#L8) to configure selector with default value.

2) postcss-mixins
Is there a way to have autocomplete for at least mantine helper functions and mixins?
Using sass I have any autocomplete for self-defined mixins.
At least for vscode I would like to see any solution.
Maybe we can control with options on `postcss-preset-mantine` which mixins we want, for example if we want for purpose of suggestions to copy-paste in css files locally.

3) suggestions for css variables
The similar question for community.
In my case I have a standalone shared theme config, used for tailwind and for mantine as well.
Most of config values have transformed in css variables through tailwind `addBase` to `:root`.
In css modules I use this variables, but don't have autocomplete, so this is hard.
In addition, when mantine go with per-component based css variables, and I have a lot of wrappers around it for v6, it would be a nightmare without suggestions.

**Comment by rtivital** [comment]
1. No, it is not possible, Mantine styles are static and do not support changing this attribute
2. There are not that much functions and mixins to think about autocomplete, most likely you will only use `rem` and `light-dark` functions in your applications
3. You can install an extension for your editor, all default Mantine css variables are in the `@mantine/core/styles.css` file
